idaes/core/solvers/tests/test_petsc.py

```diff
@@ -579,3 +636,7 @@ def test_rp_example4():
     )
     assert pyo.value(m.u[10]) == pytest.approx(398)
     assert pyo.value(m.x[10]) == pytest.approx(20)
+
+
+if __name__ == "__main__":
+    test_petsc_read_trajectory_parts()
```

Comment:
Probably remove this part in the final commit.

Reply:
Yeah.